client/main/components/program/Activity/ContentContainer.vue

```diff
@@ -0,0 +1,51 @@
+<template>
```

Comment:
This component has been moved to in `@/main/components/common/ContentContaier.vue`. Delete it from here.